x/application/keeper/query_application_test.go

```diff
@@ -1,127 +0,0 @@
-package keeper_test
```

Comment:
Where these tests moved or just not needed?

Reply:
@Olshansk bump on this confused why this is gone